app/components/homeScreen/DailyReads.tsx

```diff
@@ -307,13 +314,20 @@ const DailyReads = (): any => {
       setActivityDataToShowInList(activityDataList)
 
     }
-  }, [activeChild?.uuid, activityTaxonomyId]);
+  }, [activeChild?.uuid, activityTaxonomyId,fetchAgain]);
   const keyExtractor = useCallback((item: any) => item?.id, []);
 
   const renderItem = useCallback((item: any, index: any, isAdvice: boolean) => {
     return <RenderDailyReadItem item={item} index={index} isAdvice={isAdvice} />;
   }, [favoriteAdvices, favoriteGames]);
 
+  const renderEmptyList = () => {

```

Comment:
Function does not render anything when the list is empty, it just returns null. Maybe we should rename it to handleEmptyList

Reply:
change the name